src/components/ReportHeaderSkeletonView.tsx

```diff
@@ -48,18 +48,18 @@ function ReportHeaderSkeletonView({shouldAnimate = true, onBackButtonPress = ()
                 >
                     <Circle
                         cx="20"
-                        cy="33"
+                        cy="40"
```

Comment:
NAB: why are we adjusting each SVG component's size individually? Can't we adjust each size property in respect to the overall size of the SVG? (this might necessitate to extract the SVG to its own component, but it's just cumbersome if we have to adjust each SVG component's property individually whenever we want to change the overall size of the SVG.